README.md

```diff
@@ -115,7 +138,7 @@ The parts of this MFE that can be customized in that manner are documented [here
 
 ## Running tests
 
-1. Assuming that you're operating in the context of the edX devstack,
+1. (How to update devstack mention?) Assuming that you're operating in the context of the edX devstack,
```

Comment:
@brian-smith-tcril - what's the best way to run tests?

Reply:
`npm test`

Reply:
I honestly feel like we don't need this section. Tests can run standalone, no need to have tutor or devstack or anything set up. I guess we could put
```
nvm use
npm ci
npm test
```
in here but I really don't think it's needed.

Reply:
I added it in, I don't think it's un-useful as a reminder.